src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing/01 instrumentHTTP.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'instrumentHTTP'
+excerpt: 'instrumentHTTP instruments the k6 http module with tracing capabilities.'
+---
+
+The `instrumentHTTP` function instruments the k6 http module with tracing capabilities. It transparently replaces each of the k6 http module functions with versions that automatically attach a trace context to every request. Functions that you can instrument include [del](/javascript-api/k6-http/del), [get](/javascript-api/k6-http/get), [head](/javascript-api/k6-http/head), [options](/javascript-api/k6-http/options), [patch](/javascript-api/k6-http/patch), [post](/javascript-api/k6-http/post), [put](/javascript-api/k6-http/head), and [request](/javascript-api/k6-http/request).
```

Comment:
```suggestion
The `instrumentHTTP` function instruments the k6 http module with tracing capabilities. It transparently replaces each of the k6 http module functions with versions that automatically attach a trace context to every request. Instrumented functions include [del](/javascript-api/k6-http/del), [get](/javascript-api/k6-http/get), [head](/javascript-api/k6-http/head), [options](/javascript-api/k6-http/options), [patch](/javascript-api/k6-http/patch), [post](/javascript-api/k6-http/post), [put](/javascript-api/k6-http/head), and [request](/javascript-api/k6-http/request).
```

They are *always* instrumented all or none, so "can" here make it sound to me like you can choose which ones ... which is not true